### DISCUSSION TITLE
Is there a way to change the UI for the drop placeholder

### DISCUSSION THREAD

**Discussion by wi-mm** [GrapesJS/grapesjs#5101] [Q&A] [answered]
I am trying to customize the UI for the drop placeholder that currently shows a green border. Here is a preview of what I am referring to: https://d.pr/i/PM99J7. 

I did not find any information regarding that in the documentation. Is this even possible with the current version of GrapesJS?

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/discussions/3833

**Comment by wi-mm** [comment, marked as answer]
@artf Thank you.
